lib/apis/v3/felixconfig.go

```diff
@@ -220,6 +220,11 @@ type FelixConfigurationSpec struct {
 	// network stack is used.
 	NATPortRange *numorstring.Port `json:"natPortRange,omitempty"`
 
+	// NATOutgoingAddress specifies an address to use when performing source NAT for traffic in a natOutgoing pool that
+	// is leaving the network. By default the address used is an address on the interface the traffic is leaving on
+	// (ie it uses the iptables MASQUERADE target)
+	NATOutgoingAddress string `json:"NATOutgoingAddress,omitempty"`
```

Comment:
Did you see the thoughts from @caseydavenport at https://github.com/projectcalico/calico/issues/2222#issuecomment-484292225 ?  If we can meet the need with a global config setting for the interface name or CIDR, that would be nicer than having to set node-specific config for each node in the cluster.

Reply:
Will take a look at the missing conversion code momentarily.

Unfortunately I did not see that comment and while implementing was only considering between using a flag to use the BGP address or passing a specific IP. For us, either would work, but being able to pass a specific IP seemed more flexible.